conf/modules/sonar_adc.xml

```diff
@@ -1,15 +1,22 @@
 <!DOCTYPE module SYSTEM "module.dtd">
 
-<module name="sonar">
+<module name="sonar_adc" dir="sonar">
   <doc>
     <description>
      Sonar ADC driver.
-     Reads an anlog sonar sensor and outputs sonar distance in [m]
+     Reads an anlog sonar sensor and outputs sonar distance in [cm]
     </description>
-    <configure name="ADC_SONAR" value="ADC_X" description="select ADC to use with the sonar"/>
-    <define name="SONAR_OFFSET" value="0"  description="sensor offset in [adc] - default is 0"/>
-    <define name="SONAR_SCALE" value="0.0166" description="sensor scale factor [m/adc] - default is 0.0166"/>
-    <define name="USE_SONAR" value="" description="activate use of sonar in INS extended filter (only rotorcraft)"/>
+    <configure name="ADC_SONAR" value="ADC_X" description="Select ADC port to use with the sonar, e.g. ADC_3"/>
+    <define name="SONAR_OFFSET" value="0.0234"  description="sensor offset in [adc], default is 0"/>
+    <define name="SONAR_SCALE" value="0.0345" description="Sensor scale factor [cm/adc], default is 0.0166"/>
+    <define name="USE_SONAR" value="TRUE" description="Activate use of sonar in INS extended filter (only rotorcraft)"/>
+    <define name="SONAR_USE_ADC_FILTER" value="TRUE" description="Enable or disable the median filter on the sonor output values"/>
+    <define name="SONAR_MEDIAN_SIZE" value="9" description="If median filter is enable then set this option to filter the output more(or less), default is 7"/>
```

Comment:
there is no median filter in this driver, should be removed

Reply:
indeed atm